server/server.go

```diff
@@ -0,0 +1,43 @@
+package server
+
+import (
+	"context"
+	"fmt"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/encoding/protojson"
+
+	changelogs "github.com/crossplane/crossplane-runtime/apis/changelogs/proto/v1alpha1"
+	"github.com/crossplane/crossplane-runtime/pkg/errors"
+)
+
+type Server struct {
+	changelogs.UnimplementedChangeLogServiceServer
+}
+
+func (s *Server) SendChangeLog(ctx context.Context, req *changelogs.SendChangeLogRequest) (*changelogs.SendChangeLogResponse, error) {
```

Comment:
`SendChangeLog` is kind of a strange signature for a handler of incoming requests. 🤔 I guess it's required due to the RPC being `SendChangeLog`?

Reply:
ha it does look a bit strange on the server side, doesn't it? it makes better sense on the client side, e.g. `client.SendChangeLog()`.

But yep, it's required as part of the RPC interface, so we'll keep it here as is :)